rllib/models/torch/visionnet.py

```diff
@@ -1,5 +1,7 @@
+import numpy as np
```

Comment:
The changes in this file make sure that torch now behaves exactly the same as tf wrt Conv2D Models, which is:
- if `num_outputs`: end with Conv2D [1,1] layer with num_outputs dim.
- else: end with flatten layer with dim being determined by the last Conv2D's setup.
This allows for simple wrapping with e.g. an LSTM (or any other (custom) Model).